autotest.pm

```diff
@@ -221,6 +222,10 @@ sub loadtest ($script, %args) {
     }
     elsif ($script =~ m/\.lua$/) {
         _load_lua();    # this loads (actually defines) lua_eval()
+        unless (defined &lua_set) {
+            bmwqemu::fctwarn("Inline::Lua not present, cannot load $script");
```

Comment:
So if I read this correctly tests will just be ignored if Lua is missing. That would make it very dangerous to rely on test coverage in Lua.